edb/edgeql/codegen.py

```diff
@@ -1551,6 +1551,22 @@ def visit_SetPointerType(self, node: qlast.SetPointerType) -> None:
                 self.visit(node.cast_expr)
                 self.write(')')
 
+    def visit_SetPointerOptionality(
+        self,
+        node: qlast.SetPointerOptionality,
+    ) -> None:
+        if node.value is None:
+            self.write('RESET OPTIONALITY')
```

Comment:
Can't say I'm a big fan of this, but I see why this can be a better UX that distinct RESET REQUIRED and RESET OPTIONAL.